src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/HearingRoute.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+@Getter
+public enum HearingRoute {
+
+    @JsonEnumDefaultValue
```

Comment:
I'm not sure if we want to do this. It should be the default but only when we release S&L